search_engine_parser/core/engines/myanimelist.py

```diff
@@ -3,17 +3,25 @@
 """
 
 from search_engine_parser.core.base import BaseSearch
-from search_engine_parser.core.exceptions import NoResultsOrTrafficError
 import math
 import re
 
+"""MAL has lots of extra \n,\r etc. Need to remove them"""
+def style_format(text):
+	
```

Comment:
Why this function?

Reply:
It's important because as I mentioned, there are lots of extra characters in MyAnimeList descriptions.
Right now, the returned list for ratings is something like:
```'
   ratings': ['7.92',
             '8.20',
             '7.51',
             '7.24',
             '7.40',
             '7.87',
             '7.40',
             '7.52',
             '7.75',
             '6.53']
```
while **without** the function, it would be:

```
 'ratings': ['\n    7.92\n  ',
             '\n    8.20\n  ',
             '\n    7.51\n  ',
             '\n    7.24\n  ',
             '\n    7.40\n  ',
             '\n    7.87\n  ',
             '\n    7.40\n  ',
             '\n    7.52\n  ',
             '\n    7.75\n  ',
             '\n    6.53\n  ']
```
which in my opinion is just plain ugly and unnecessary. Same is the case for other bunches of text. Thus, I believe this extra formatting would be visually pleasing to the users/easier to use in case they write their wrappers over this library.

I would certainly be willing to remove it in case I am missing a major reason why it shouldn't be there 😄

Reply:
Why not just strip it `.strip()`?

Reply:
I was being extra cautious as ```.strip()``` only removes extra characters from beginning and start, but the function I wrote would have done it from all places. However, on experimenting, it seems that your point is valid and it won't be necessary as the extra characters are at beginning/end only. I've removed it